pkg/kudoctl/kudoinit/crd/bindata.go

```diff
@@ -96,7 +96,7 @@ func configCrdsKudoDev_instancesYaml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "config/crds/kudo.dev_instances.yaml", size: 1040, mode: os.FileMode(420), modTime: time.Unix(1576836957, 0)}
+	info := bindataFileInfo{name: "config/crds/kudo.dev_instances.yaml", size: 0, mode: os.FileMode(0), modTime: time.Unix(0, 0)}
```

Comment:
hmm I don't understand why were these changes required, can you explain please?

Reply:
generated files should not change if the source does not change... the way bindata.go was originally implemented (which was recent) seems less mature than it should be in a number of ways.  this pr resolves that.   It isn't ok to have a dirty local repo with a `make all` or `make generate`.    After a couple of weeks of wonder what was the issue... it was this.

it is the timestamp that is most offensive... it will change based on a new clone of the repo etc.  There is a flag to "set" modtime however it doesn't take a `0` or it appears that 0 means generate it.  we could set it to `1` but I thought that would be confusing.  specifying "no metadata" zeros out modtime... and everything else.   It isn't particular useful information for our context... and in most likely case, it was accidental... meaning it wasn't a decision... it was the default behavior.

Reply:
oh I see, so the key is `-nometadata`. Yeah I did not get that part. Thanks for the explanation

Reply:
thx, for the explanation, @kensipe , I was wondering the same thing as Alena